examples/cfd/01_convection_revisited.ipynb

```diff
@@ -486,7 +491,25 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.2"
+   "version": "3.6.9"
```

Comment:
what is this stuff? they need to go

Reply:
I don't control what Jupiter does and not gonna reinstall a new version of python.

Reply:
I typically use vim to manually edit the cells that need to be edited, so that we don't get this garbage. However, I understand it's a pain... so never mind